installation/upgrade-to-the-latest-version.md

```diff
@@ -9,11 +9,12 @@ date: 17/5/2017
 This article explains how to upgrade the SPDocKit to the latest major version. SPDocKit database, saved farms, snapshots and application settings will be conserved in the upgrade process.
 
 ## Preparation:
-1. The account running the application should have __db_owner__ privileges on SPDocKit database to perform the upgrade process successfully. In case that you are also changing the SPDocKit Service account to the different one from the Service account that has been assigned to the previous application version, account running the upgrade should also have __securityadmin__ role on the preferred SQL Server. This role will allow him to [grant proper privileges to the new Service account](#internal/requirements/spdockit-user-permission-requirements/).
+1. The account running the application should have __db_owner__ privileges on SPDocKit database to perform the upgrade process successfully. In case that you are also changing the SPDocKit Service account to the different one from the Service account that has been assigned to the previous application version, account running the upgrade should also have __securityadmin__ role on the preferred SQL Server. This role will allow him to [grant proper privileges to the new Service account](#internal/requirements/user-permission-requirements.md/).
 2. Perform the __database backup__ before proceeding with the upgrade process, especially to the latest versions of the SPDocKit.
-3. Due to a number of database modifications, upgrade to SPDocKit 6.0 and 7.0 may take a while. Also, [License Management](#internal/get-to-know-documentation-toolkit/backstage-screen/license-management/) configuration will not be preserved.
-4. We redesigned email alerts feature, previously located under the Email Alerts tab of Options screen. __Email alerts will not be preserved__ when upgrading to the version 6.2.0. and later. You will have to create new alerts. For more information on this, please follow [this](#internal/get-to-know-documentation-toolkit/backstage-screen/options-wizard/#email-alerts)  instructions.
+3. Due to a number of database modifications, upgrade to SPDocKit 6.0 and 7.0 may take a while. Also, [License Management](#internal/get-to-know-spdockit/backstage-screen/license-management.md/) configuration will not be preserved.
+4. We redesigned email alerts feature, previously located under the Email Alerts tab of Options screen. __Email alerts will not be preserved__ when upgrading to the version 6.2.0. and later. You will have to create new alerts. For more information on this, please follow [this](#internal/get-to-know-spdockit/backstage-screen/options-wizard.md#email-alerts)  instructions.
 5. If you are upgrading from an SPDocKit version lower than 5.1.0., make sure you are on the SharePoint farm you were using this database for. Since the latest upgrade process uses a FarmID for data classification, upgrading SPDocKit database on other farms might result in data corruption.
```

Comment:
uses __the__ farm ID
upgrading __the__ SPDocKit database on other farms might result in data corruption.